packages/client/tests/functional/jest.config.js

```diff
@@ -25,4 +25,22 @@ module.exports = {
   setupFilesAfterEnv: ['./_utils/setupFilesAfterEnv.ts'],
   testTimeout: 10000,
   collectCoverage: process.env.CI ? true : false,
+
+  /**
+   * Limit concurrency to 1 worker for now for two reasons.
+   *
+   * First reason:
+   *
+   * Database isolation in setupTestSuiteDbURI() doesn't seem to work properly.
+   * Tests are using to the same database concurrently, which causes constraint
+   * violation errors and "database does not exist" errors.
+   * TODO: fix this.
+   *
+   * Second reason:
+   *
+   * Jest crashes when an assertion fails with an object that contains a BigInt
+   * if more than one worker is used.
+   * See https://github.com/facebook/jest/issues/11617
```

Comment:
Let's +1 this and also leave a comment that we are also affected to raise awareness a bit.